kde-apps/konsole/konsole-22.04.2.ebuild

```diff
@@ -17,6 +17,7 @@ LICENSE="GPL-2" # TODO: CHECK
 SLOT="5"
 KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86"
 IUSE="X"
+PATCHES=( "${FILESDIR}"/${PN}-22.04.2-musl_malloc_trim.patch )
```

Comment:
```suggestion

PATCHES=( "${FILESDIR}"/${PN}-22.04.2-musl_malloc_trim.patch )
```

Reply:
That's not where we put the PATCHES array. Please fix it.

Reply:
Ugh, sorry.